.github/workflows/renovate_linting.yml

```diff
@@ -0,0 +1,12 @@
+name: Renovate Config Linting
```

Comment:
Recommend renaming this file to `renovate-lint.yml` to comply with existing workflow naming conventions: https://github.com/wayfair-incubator/expirable-storage/tree/main/.github/workflows

Reply:
Thanks and recommendation is adapted in the updated commit.